<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameForm</class>
 <widget class="QWidget" name="GameForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Имитация игры</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QGroupBox" name="groupBoxHost">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>685</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:White;</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="labelAlivePlayers">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>10</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelHost">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>471</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>(Ведущий говорит громко)</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxPlayers">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>681</width>
     <height>350</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: Maroon;
border: 3px groove blue;
border-radius: 5px</string>
   </property>
   <property name="title">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
